src/Host/Client/Impl/Extensions/RStringExtensions.cs

```diff
@@ -91,5 +91,13 @@ public static class RStringExtensions {
             }
             return s;
         }
+
+        public static string ToRPath(this string s) {
```

Comment:
I'm curious about what happens if packages are installed under the user directory. Normally, R shortens such paths using `~` - if that is also the case here, it needs to be dealt with. We have `MakeRelativeToRUserDirectoryAsync` method which may be of help.